sys/libkern/memset_c.c

```diff
@@ -0,0 +1,32 @@
+/*-
+ * Copyright (c) 2014 SRI International
```

Comment:
This probably isn't the right copyright and sponsor ACK?

Reply:
I copied the file from libc, so I guess it is?